app/pdfJS.js

```diff
@@ -5,23 +5,102 @@
 
 const UrlUtil = require('../js/lib/urlutil')
 const Filtering = require('./filtering')
-const config = require('../js/constants/config')
 const appActions = require('../js/actions/appActions')
 const getSetting = require('../js/settings').getSetting
 const settings = require('../js/constants/settings')
 
-const pdfjsBaseUrl = `chrome-extension://${config.PDFJSExtensionId}/`
-const viewerBaseUrl = `${pdfjsBaseUrl}content/web/viewer.html`
+const getViewerUrl = UrlUtil.getPDFViewerUrl
+
+/**
+ * Check if the request is a PDF file.
+ * @param {Object} details First argument of the webRequest.onHeadersReceived
+ *                         event. The properties "responseHeaders" and "url"
+ *                         are read.
+ * @return {boolean} True if the resource is a PDF file.
+ */
+function isPdfFile (details) {
```

Comment:
To match other places, this (and the function below) might be named with the same casing (ex: `isPDFFile`). If the functions aren't being exported, they could be changed to `isFile` and `isDownloadable` since this file is already named `pdfJS.js`